### ISSUE TITLE
Robolectric test application setup throws IllegalArgumentException, loading same font twice

### ISSUE THREAD

**Issue by andrewsardone** [JoanZapata/android-iconify#114]
This is more of a question / out-loud-thinking to see if anyone has come across this issue, or if anyone sees any obvious mistakes I have made.

I have a simple `android.app.Application` subclass which registers my `Iconify` modules within the `onCreate()` implementation (per the [README instructions](https://github.com/JoanZapata/android-iconify/blob/1515366e5e0dda9a666bc32f22c0e94f6804c147/README.md#install)). Everything works and runs fine in a normal context, but all of my JUnit tests (ran with [Robolectric](https://github.com/robolectric/robolectric/blob/bf30ae3c6f2b7da68f30bd05d6151cd384f7fcce/README.md#usage)) are throwing exceptions:

```
Caused by: java.lang.IllegalArgumentException: Can't add twice the same font "iconify/android-iconify-fontawesome.ttf"
	at com.joanzapata.iconify.Iconify.addIconFontDescriptor(Iconify.java:45)
	at com.joanzapata.iconify.Iconify.access$000(Iconify.java:11)
	at com.joanzapata.iconify.Iconify$IconifyInitializer.<init>(Iconify.java:68)
	at com.joanzapata.iconify.Iconify.with(Iconify.java:22)
	at com.example.FooApplication.onCreate(FooApplication.java:51)
	at org.robolectric.internal.ParallelUniverse.setUpApplicationState(ParallelUniverse.java:126)
	at org.robolectric.RobolectricTestRunner.setUpApplicationState(RobolectricTestRunner.java:440)
	at org.robolectric.RobolectricTestRunner$2.evaluate(RobolectricTestRunner.java:222)
	... 35 more
```

Has anyone come across this? Any ideas on how to fix the issue? Apologies for the lack of details; just not sure where to go next. Thanks!

**Comment by andrewsardone** [comment]
It's far from a solution, but my current hack is to [key off of a static flag for whether or not we've loaded the module](https://gist.github.com/andrewsardone/0ccc3a934ecc4841752b) :shame:

**Comment by JoanZapata** [comment]
Oh.

I throw this exception to prevent people from initializing Iconify in every activity instead of just once in the application. I didn't think about tests instantiating multiple applications. I'll change this behavior in the next release, adding twice the same font would do.. well, nothing.

**Comment by JoanZapata** [comment]
Fixed in 2.0.9 released just now.
